src/frontend/magnify/src/components/design-system/Formik/LanguageChange/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { useFormikContext } from 'formik';
+import * as React from 'react';
+import { PropsWithChildren, useEffect } from 'react';
+import { useLocale } from '../../../../i18n/TranslationProvider/TranslationsProvider';
+
+export const FormikLanguageChange = ({ children }: PropsWithChildren) => {
```

Comment:
What do you think to call this component `LocalizedForm`  then instead of simply return `children` it returns `<Form>{children}</Form>` ?